spec/controllers/api/v1/items_controller_spec.rb

```diff
@@ -0,0 +1,13 @@
+RSpec.describe Api::V1::ItemsController do
```

Comment:
We should get together and talk about how we're writing our controller tests --- update them according to today's lesson